src/gdcmodels/esmodels/gdc_from_graph/descriptions.yaml

```diff
@@ -87,8 +87,8 @@ annotations.analyte.submitter_id: A project-specific identifier for a node. This
   of the uuid for identifying or recalling a node.
 annotations.analyte.updated_datetime: A combination of date and time of day in the
   form [-]CCYY-MM-DDThh:mm:ss[Z|(+|-)hh:mm]
-annotations.analyte.well_number: Numeric value that represents the the well location
-  within a plate for the analyte or aliquot from the sample.
+annotations.analyte.well_number: Numeric value that represents the well location within
```

Comment:
I'm going to assume these additions are all correct. The ole gdc-models have fallen behind a bit.

Reply:
You can def double check the data models to be sure. I did spot check but that obviously isn't 100%. E.g. specimen_type is a field that was added after we stopped updating this file with the output of esbuild. Other places I mainly see typo fixes etc.